flocker/node/agents/ebs.py

```diff
@@ -29,6 +32,45 @@
 CLUSTER_ID_LABEL = u'flocker-cluster-id'
 ATTACHED_DEVICE_LABEL = u'attached-device-name'
 BOTO_NUM_RETRIES = u'10'
+BOTO_DEBUG_LEVEL = u'1'
```

Comment:
:question: Maybe add a comment and a link explaining what this will level means:
 * https://code.google.com/p/boto/wiki/BotoConfig#Boto